fhir-persistence/src/test/java/com/ibm/fhir/persistence/test/common/AbstractCompartmentTest.java

```diff
@@ -95,38 +109,45 @@ private Reference buildReference(Resource resource) {
 
     @Test
     public void testPatientCompartment() throws Exception {
-        List<Resource> results = runQueryTest("Patient", savedPatient.getId(), 
+        List<Resource> results = runQueryTest("Patient", savedPatient.getId(),
                                     Observation.class, "_id", savedObservation.getId());
         // This currently returns 2 due to https://github.com/IBM/FHIR/issues/265
 //        assertEquals(1, results.size());
         assertTrue(results.size() > 0);
```

Comment:
not really related to you changes, but I see 265 is closed now. maybe remove the comment and uncomment the assert?
```suggestion
        assertEquals(1, results.size());
```

Reply:
Done.